src/zope/meta/multi_call.py

```diff
@@ -38,24 +39,26 @@
 args, sub_args = parser.parse_known_args()
 packages = list_packages(args.packages_txt)
```

Comment:
I think these two lines should be part of main() too.

Reply:
I moved all the argparse bits into the respective `main` function